media-libs/libxmp/libxmp-4.6.1.ebuild

```
@@ -0,0 +1,28 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Library that renders module files to PCM data"
+HOMEPAGE="https://github.com/libxmp/libxmp"
+
+if [[ ${PV} == *9999 ]] ; then
+	inherit autotools git-r3
+	EGIT_REPO_URI="https://github.com/libxmp/libxmp.git"
+else
+	SRC_URI="https://github.com/libxmp/${PN}/releases/download/${P}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~loong ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+# bzip2 depacker code is 0BSD
+LICENSE="LGPL-2.1+ MIT 0BSD"
```

```suggestion
LICENSE="MIT 0BSD public-domain"
```
Please update this also in the live ebuild.